### PR TITLE
Flutter Conf India 2023

### PR DESCRIPTION
**Pull request by ueman**
There's no dates or other information yet for Flutter Conf India, so this is a draft for the time being

### CODE REVIEW

File: _conferences/flutter-conf-india-2023.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+location: India
+online: false
+
+date_start: ???
+date_end:   ???
+
+cfp:
+  start: ???
+  end:   ???
+  site: https://flutterconf.in/
```
Comment (marandaneto) [inline _conferences/flutter-conf-india-2023.md]: https://x.com/flutterconfin/status/1699039378046398585
Maybe there's a date already? On my phone so didn't find it easily

Comment (ueman) [reply _conferences/flutter-conf-india-2023.md]: I didn't find any on their webpage or their twitter posts :/

Comment (marandaneto) [reply _conferences/flutter-conf-india-2023.md]: @ueman Dec the 9th

Comment (ueman) [reply _conferences/flutter-conf-india-2023.md]: I'm currently on vacation without a computer. Feel free to edit and merge yourself :)

Comment (marandaneto) [reply _conferences/flutter-conf-india-2023.md]: Enjoy OOO 🥳

**Review by marandaneto — APPROVED** [review]

**Pull request merged**